Coursera form

PranavBakre opened the pull request.

PranavBakre committed:
test

PranavBakre committed:
Kuch to hai XD

PranavBakre committed:
Removed commented section that isnt needed anymore

PranavBakre committed:
added panel/div field

PranavBakre committed:
changed year from textbox to dropdown

anunaym14 commented:
Form isn't centered?

PranavBakre committed:
polytechnic

PranavBakre committed:
polytechnic added in schools

PranavBakre committed:
polytechnic option added in select

PranavBakre committed:
replaced option fields for faculty with script

anunaym14 commented:
> Form isn't centered?

@akhilnarang @PranavBakre

PranavBakre commented:
@anunaym14 Wait what

PranavBakre commented:
@PranavBakre Its coming centered on mine

anunaym14 commented:
![csf](https://user-images.githubusercontent.com/27849919/79695115-794ac200-8292-11ea-877f-1143e69671f9.png)
![csf-expanded](https://user-images.githubusercontent.com/27849919/79695117-7bad1c00-8292-11ea-81f5-74623b976b84.png)
@PranavBakre

anunaym14 commented:
Probably that resolution thing again?

PranavBakre commented:
@anunaym14 no. more like thats not full form. It opens after you select faculty

anunaym14 commented:
It isn't centered even when the form is expanded..see the screenshots

parikshitgupta1 reviewed (dismissed):
add the logo of SCRIPT, Coursera, MIT-WPU as well

> > > It isn't centered even when the form is expanded..see the screenshots
> > 
> > 
> > In my browser, it is centered.
> 
> You can't ship software that works fine on your machine :)

I guess it's working fine over @PranavBakre  system as well. Not considering my case *FYI*  ^_^

anunaym14 commented:
Also @PranavBakre, add restriction to Panel/Division, so as to ensure adequate info

parikshitgupta1 commented:
> It isn't centered even when the form is expanded..see the screenshots

In my browser, it is centered.

anunaym14 commented:
> > It isn't centered even when the form is expanded..see the screenshots
> 
> In my browser, it is centered.

You can't ship software that works fine on your machine :)

PranavBakre commented:
@anunaym14 then thats because of the div thats above the form. vertical center jugaad

PranavBakre commented:
@PranavBakre div above the form is of 10vh height, which looks more or less centered on 768p i guess. 1080p and above ka kuch dekhna hoga

PranavBakre committed:
Email and phone

anunaym14 commented:
@PranavBakre yes the expanded view is centered now, can we do the initial form to be centered too and then expanding in both directions to make expanded view also centered

anunaym14 commented:
@akhilnarang is there any resource limitation from netlify?

anunaym14 commented:
Also, TODO : Form validation

akhilnarang commented:
@anunaym14 what kind of limitation, none that I know of

ksdfg commented:
@PranavBakre @anunaym14 How much of this is working? trying to shift to materialize, no faculty options....

anunaym14 commented:
Yeah @ksdfg something is wrong with the current one too, programs don't load sometimes

anunaym14 commented:
> @anunaym14 what kind of limitation, none that I know of
Idk.. pranav says form is completely fine on local, program options don't load sometimes on netlify

PranavBakre commented:
@anunaym14 Ksdfg issue has been solved

anunaym14 commented:
> @anunaym14 Ksdfg issue has been solved

Great

ksdfg commented:
ummm, what name should be given to phone number field for hades to handle? cuz currently it's email lol @PranavBakre 

Also what about validations?

PranavBakre commented:
oh lol. I didnt see that.

for validation, i had asked on tg about that if someone could do it.

We cant validate year till we are able to impose limits, phone number length can be validated

PranavBakre commented:
ERP ID; we'd need password for using poseidon to validate that

PranavBakre commented:
For division, same as year; we need to know actual limits

ksdfg committed:
material css ftw

ksdfg commented:
ok, b4 anyone curses me for the bg, I assume something good is being made. This is just a stand-in.

anunaym14 commented:
> ok, b4 anyone curses me for the bg, I assume something good is being made. This is just a stand-in.

not sure if something is being done, better to have something usable before-hand

ksdfg committed:
add validation for empty faculty

akhilnarang commented:
@PranavBakre For ERP ID, 2 things

1. Rename to PRN just to be inline with other tables in our database
2. Maxlength 11, prefill S1032 for them and either add custom checks or just a pattern matching S\d+

ksdfg commented:
> @PranavBakre For ERP ID, 2 things
> 
>     1. Rename to PRN just to be inline with other tables in our database
> 
>     2. Maxlength 11, prefill S1032 for them and either add custom checks or just a pattern matching S\d+

do the maxlength thing yourself, I'll change name

ksdfg committed:
stuff

ksdfg committed:
meow stuff

ksdfg committed:
panties slide down

PranavBakre committed:
meta viewport

ksdfg committed:
set width with vw, with max value

AniketRaj314 committed:
Ph.d -> Ph,D

AniketRaj314 committed:
Ph.d -> Ph,D

AniketRaj314 committed:
Added mail, responsive af

AniketRaj314 committed:
Merge branch 'CourseraForm' of https://github.com/The-SCRIPT-Group/Website into CourseraForm

akhilnarang commented:
Someone see about AJAX call and rest of the validations

ksdfg commented:
@akhilnarang gayjax

ksdfg commented:
@ksdfg didn't know you can send comments from tele, Imma do this all the time now

anunaym14 commented:
> @ksdfg didn't know you can send comments from tele, Imma do this all the time now

Lmao

PranavBakre committed:
@ksdfg1247 gonna kill me XD

PranavBakre committed:
changed background resolution from 800p to 768p

added logos

modified CSS

decreased button size

removed non working vertical center jugaad

PranavBakre committed:
test for background image being zoomed on full ofrm open

PranavBakre committed:
added onchange for school for selecting program per school.
removed console.log() from Coursera.js

PranavBakre committed:
phone and erp id validation. hopefully correct

PranavBakre committed:
PRN without S:

adityavdesai committed:
Fix input background with autocomplete

adityavdesai committed:
Remove Panel

adityavdesai committed:
Enter full name beech

PranavBakre committed:
slide up on prn

PranavBakre committed:
formatting coursera.html

PranavBakre committed:
tried to write ajax code

PranavBakre committed:
some changes

PranavBakre committed:
removed duplicate on submit

PranavBakre committed:
test

PranavBakre committed:
form closes after reset

ksdfg committed:
so shit breaks if I'm not there.... fixing error message and useless import

PranavBakre committed:
modal and loader

PranavBakre committed:
no qr

akhilnarang committed:
coursera: formatting, remove some newlines

Signed-off-by: Akhil Narang <me@akhilnarang.dev>

akhilnarang committed:
coursera: add mail contents, I guess

  Thanks kursiwala for `nice_email.html`

Signed-off-by: Akhil Narang <me@akhilnarang.dev>

akhilnarang committed:
coursera: formatting

Signed-off-by: Akhil Narang <me@akhilnarang.dev>

akhilnarang committed:
coursera: fix variable name

Signed-off-by: Akhil Narang <me@akhilnarang.dev>

akhilnarang committed:
coursera: rip these field names

Signed-off-by: Akhil Narang <me@akhilnarang.dev>

rittmang committed:
Remove button from email content
